README.md

```diff
@@ -61,10 +44,10 @@ LOCAL_MODEL_PATH is required. The two AWS access variables are required unless s
 
 To debug your configuration, try opening a Python shell and run `from spacer import config`, then `config.check()`.
 
-#### Docker build
-The docker build is the preferred build and the one used in deployment.
+### Docker build
+The docker build is used in coralnet's deployment.
 * Install docker on your system
-* Create `secrets.json` as detailed above.
+* Set up configuration as detailed above.
 * Create folder `/path/to/your/local/models` for caching model files.
 * Build image: `docker build -t spacer:test .`
```

Comment:
Why the `test` tag?

Reply:
I guess that doesn't have to be in these instructions. It's just that the CoralNet process for building and deploying Docker images involves tagging them.

Reply:
See commit 6682a39 - generalized `spacer:test` to `myimagename`, and tried to clarify some surrounding instructions too.